src/components/FormToggle.vue

```diff
@@ -8,6 +8,15 @@
     >
         <slot name="left"></slot>
         <input
+            v-if="showFalseInput"
+            :name="name"
+            :value="falseValue"
+            :disabled="disabled"
+            type="hidden"
+            @change.stop="toggle"
```

Comment:
why stop? 

emit also change event like in:
https://github.com/odyzeo/form-checkbox/blob/master/src/components/FormCheckbox.vue#L113

+ add readonly prop w/ same logic as in checkbox please, it might be useful later